README.md

```diff
@@ -46,13 +55,18 @@ The nf-core/bcellmagic pipeline comes with documentation about the pipeline: [us
 
 nf-core/bcellmagic was originally written by Gisela Gabernet, Simon Heumos, Alexander Peltzer.
 
+We thank the following people for their extensive assistance in the development
+of this pipeline:
+
+<!-- TODO nf-core: If applicable, make list of people who have also contributed -->
```

Comment:
Include acknowledgements to all people who were / are involved :-)

Reply:
Here is still mostly the two of us and Simon, we are already mentioned as main pipeline developers above.